### PR TITLE
cleanup and reformat bash files at repositor root

### PR DESCRIPTION
**Pull request by tuminoid**
It is time to make dev-env more consistent as we see plenty of changes happening here.

Make all root level bash files look the same. Same indent, same curlys, same nice bash syntax all around.

0* scripts will be fixed in follow-up, and libs after that.

### CODE REVIEW

**Comment by tuminoid** [comment]
/test metal3-dev-env-integration-test-ubuntu-main metal3-centos-e2e-integration-test-release-1-8
Unrelated fail.

**Comment by dtantsur** [comment]
In an ideal world, we would rewrite this all in something like ansible.. we're not in an ideal world, and the change looks good so far.

**Comment by tuminoid** [comment]
> In an ideal world, we would rewrite this all in something like ansible.. we're not in an ideal world, and the change looks good so far.

Yes, there are ideas how to improve (in my ideal world it is not ansible though :laughing:), but since we're seeing lots of changes coming here, it'll make reviewing easier to have all these bash monsters at least looking alike.

PTAL.
/cc @dtantsur @lentzi90 @smoshiur1237

**Comment by smoshiur1237** [comment]
Changes are looking good. I will check one more time

**Review by smoshiur1237 — COMMENTED** [review]
Thanks for the improvement. 
/approve

File: host_cleanup.sh [hunk 1]
```diff
@@ -55,26 +56,26 @@ ANSIBLE_FORCE_COLOR=true "${ANSIBLE}-playbook" \
...
-    sudo nmcli con delete "${BARE_METAL_PROVISIONER_INTERFACE}"
-    sudo nmcli con delete provisioning
-  fi
-  # External net should have been cleaned already at this stage, but we double
-  # check as leaving it around causes issues when the host is rebooted
-  if [[ "${MANAGE_EXT_BRIDGE}" == "y" ]]; then
-    sudo nmcli con delete external || true
-  fi
+    sudo rm -rf /etc/NetworkManager/conf.d/dnsmasq.conf
+    if [[  "${MANAGE_PRO_BRIDGE}" == "y" ]]; then
```
Comment (dtantsur) [inline host_cleanup.sh:60]: nit: redundant space after [[

**Comment by dtantsur** [comment]
I don't think I have any rights here but

/lgtm

**Pull request merged**